.github/workflows/release.yml

```diff
@@ -0,0 +1,93 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - alpha
+      - beta
```

Comment:
Would it make sense to `npx wrangler` after a successful release?

Reply:
Tried that in https://github.com/open-sauced/catsup-app/pull/2/commits/5f3b486600a163ed513f34d44668f41efcb2ae69 but it leaks deployment URL :/ can put it back and test it out for a while